RomanASP.py

```diff
@@ -396,15 +408,17 @@ def main():
                 X = np.zeros_like(X)
                 X[106] = f
 
+        # Saving the output. The output needs two sections, one where we
+        # create a lightcurve compared to true values, and one where we save
+        # the images.
 
-        #Saving the output. The output needs two sections, one where we create a lightcurve compared to true values, and one where we save the images.
-        
         if use_real_images:
             identifier = str(ID)
-            lc = build_lightcurve(ID, exposures, sn_path, confusion_metric, detim, X, use_roman, band, object_type)
+            lc = build_lightcurve(ID, exposures, sn_path, confusion_metric, \
```

Comment:
I would suggest taking the opportunity here to calculate flux here, just like you do sigma_flux and then pass each to `build_lightcurve`.  It's a little non-parallel to have `X`, which `build_lightcurve` processes, and then `sigma_flux`, which `build_lightcurve` just accepts.

This will also simplify the calling sequence as you won't need to path `detim`, which I think is only used in `build_lightcurve` to specify the number of images to index lightcurve flux points from `X`.